sensor/common/detector/enricher.go

```diff
@@ -279,8 +279,7 @@ func (e *enricher) getImages(deployment *storage.Deployment) []*storage.Image {
 
 		// This will ensure that when we change the Name of the image
 		// that it will not cause a potential race condition
-		// cloning the full object is too expensive and also unnecessary
-		image := *imgResult.image
```

Comment:
@lvalerom @fredrb I remember, we got rid of some cloning to address the memory issues for Amadeus. Not sure if it was here. Could you check this please?

Reply:
This code is from 4 years ago. 😅